src/scripts/items/PokemonItem.ts

```diff
@@ -43,3 +56,7 @@ ItemList['Burmy (plant)']        = new PokemonItem(GameConstants.PokemonItemType
 ItemList['Cherubi']              = new PokemonItem(GameConstants.PokemonItemType.Cherubi, 5000);
 ItemList['Spiritomb']            = new PokemonItem(GameConstants.PokemonItemType.Spiritomb, 5000);
 ItemList['Meloetta (pirouette)'] = new PokemonItem(GameConstants.PokemonItemType['Meloetta (pirouette)'], 50000);
+
+ItemList['Deoxys (attack)']      = new PokemonItem(GameConstants.PokemonItemType['Deoxys (attack)'], 500, GameConstants.Currency.battlePoint, new ObtainedPokemonRequirement(pokemonMap.Deoxys));
+ItemList['Deoxys (defense)']     = new PokemonItem(GameConstants.PokemonItemType['Deoxys (defense)'], 500, GameConstants.Currency.battlePoint, new ObtainedPokemonRequirement(pokemonMap.Deoxys));
+ItemList['Deoxys (speed)']       = new PokemonItem(GameConstants.PokemonItemType['Deoxys (speed)'], 500, GameConstants.Currency.battlePoint, new ObtainedPokemonRequirement(pokemonMap.Deoxys));
```

Comment:
Do you think 1000 would be too expensive?
```suggestion
// Battle Frontier
ItemList['Deoxys (attack)']  = new PokemonItem(GameConstants.PokemonItemType['Deoxys (attack)'], 1000, GameConstants.Currency.battlePoint, new ObtainedPokemonRequirement(pokemonMap.Deoxys));
ItemList['Deoxys (defense)'] = new PokemonItem(GameConstants.PokemonItemType['Deoxys (defense)'], 1000, GameConstants.Currency.battlePoint, new ObtainedPokemonRequirement(pokemonMap.Deoxys));
ItemList['Deoxys (speed)']   = new PokemonItem(GameConstants.PokemonItemType['Deoxys (speed)'], 1000, GameConstants.Currency.battlePoint, new ObtainedPokemonRequirement(pokemonMap.Deoxys));
```

Reply:
Works for me. The 500 BP cost was semi-arbitrary - I was just going off the initial master ball cost as a benchmark for "expensive, but not crazy expensive".